certif/tests/integration/components/enrolled-candidates-test.js

```diff
@@ -112,9 +113,40 @@ module('Integration | Component | enrolled-candidates', function(hooks) {
     `);
 
       if (buttonVisible) {
-        assert.dom(`.${ADD_BUTTON_SELECTOR}`).isVisible();
+        assert.dom(`.${ADD_SINGLE_CANDIDATE_BUTTON_SELECTOR}`).isVisible();
       } else {
-        assert.dom(`.${ADD_BUTTON_SELECTOR}`).isNotVisible();
+        assert.dom(`.${ADD_SINGLE_CANDIDATE_BUTTON_SELECTOR}`).isNotVisible();
+      }
+    }),
+  );
+
+  [
+    { isSco: true, toggle: true, buttonVisible: true },
+    { isSco: true, toggle: false, buttonVisible: false },
+    { isSco: false, toggle: false, buttonVisible: false },
+    { isSco: false, toggle: true, buttonVisible: false },
+  ].forEach(({ isSco, toggle, buttonVisible }) =>
```

Comment:
Les deux boutons sont exclusifs, on pourrait merger les 2 tests en 1 seul:

```suggestion
  [
    { isSco: true, toggle: true, buttonSingleVisible: true },
    { isSco: true, toggle: false, buttonSingleVisible: false },
    { isSco: false, toggle: false, buttonSingleVisible: false },
    { isSco: false, toggle: true, buttonSingleVisible: false },
  ].forEach(({ isSco, toggle, buttonVisible }) =>
```
Si on buttonSingleVisible === true => on verifie qu'il est affiché et pas le boutton multiple
Si on buttonSingleVisible === false => on verifie qu'il n'est pas affiché et mais que le  boutton multiple oui